test/mailers/previews/bounty_mailer_preview.rb

```diff
@@ -0,0 +1,8 @@
+# Preview all emails at http://localhost:3000/rails/mailers/bounty_mailer
+class BountyMailerPreview < ActionMailer::Preview
+  def open_bounty
+    BountyMailer.with(user: User.first,
+                      bounties: Bounty.where(status: "open").last(15)).open_bounty
+  end
+
```

Comment:
This isn't a great example.  The data we're passing into it depends on there already being User and Bounty objects in our database.  That means this will fail if we don't have our DB set up correctly.  Usually, I'll use fixtures of the object I need in the mailers to make sure they're always there and available to use in the view.